awsagentprovider/src/test/java/software/amazon/opentelemetry/javaagent/providers/AwsMetricAttributeGeneratorTest.java

```diff
@@ -1029,6 +1030,38 @@ private void validateHttpStatusForNonLocalRootWithThrowableForClient(
     }
   }
 
+  @Test
+  void testDBUserAttribute() {
+    String dbUser = "test_user";
+    mockAttribute(DB_USER, dbUser);
+    when(spanDataMock.getKind()).thenReturn(SpanKind.CLIENT);
```

Comment:
Please add a test for `SpanKind.SERVER` that asserts `AWS_DB_USER` attribute not present in `SERVICE_METRIC`